core/block/import/source/source.go

```diff
@@ -15,7 +15,10 @@ var log = logging.Logger("import-source")
 var extensions = []string{".md", ".csv", ".txt", ".pb", ".json", ".html"}
 
 type Source interface {
-	GetFileReaders(importPath string, ext []string, includeFiles []string) (map[string]io.ReadCloser, error)
+	Initialize(importPath string) error
```

Comment:
In `Initialized` we collect all files from given resource (zip archive, directory or simple file) and save them in `map`